shell/platform/tizen/tizen_view_elementary.h

```diff
@@ -58,6 +60,7 @@ class TizenViewElementary : public TizenView {
   std::vector<Ecore_Event_Handler*> ecore_event_key_handlers_;
 
   bool scroll_hold_ = false;
+  bool has_focus_ = false;
```

Comment:
How about `focused`?
As far as I know, `has_focus` is usually used when there is a focused sub-element.
ex)
parent (`focused` and `has_focus`) - child
/
parent (`has_focus`) - child (`focused`)